mono/tests/Makefile.am

```diff
@@ -1078,13 +1129,21 @@ TESTSAOT_IL=$(TESTS_IL:.exe=.exe$(PLATFORM_AOT_SUFFIX))
 TESTSAOT_BENCH=$(TESTS_BENCH:.exe=.exe$(PLATFORM_AOT_SUFFIX))
 TESTSAOT_STRESS=$(TESTS_STRESS:.exe=.exe$(PLATFORM_AOT_SUFFIX))
 TESTSAOT_GSHARED=$(TESTS_GSHARED:.exe=.exe$(PLATFORM_AOT_SUFFIX))
+TESTSAOT_TAILCALL=$(TESTS_TAILCALL_COMPILE:.exe=.exe$(PLATFORM_AOT_SUFFIX))
 endif
 
 EXTRA_DIST=test-driver test-runner.cs $(TESTS_CS_SRC) $(TESTS_IL_SRC) \
-	$(TESTS_BENCH_SRC) $(TESTS_STRESS_SRC) stress-runner.pl
+	$(TESTS_BENCH_SRC) $(TESTS_STRESS_SRC) stress-runner.pl \
+	$(TEST_TAILCALL_CS_SRC) \
+	$(TEST_TAILCALL_IL_SRC)
+
+# This test has extra sources.
+tailcall/coreclr/JIT/opt/Tailcall/TailcallVerifyWithPrefix.exe: \
+		tailcall/coreclr/JIT/opt/Tailcall/TailcallVerifyTransparentLibraryWithPrefix.il \
+		tailcall/coreclr/JIT/opt/Tailcall/TailcallVerifyVerifiableLibraryWithPrefix.il
 
 %.exe: %.il
-	$(ILASM) -out:$@ $<
+	$(ILASM) -out:$@ $^
```

Comment:
Why?

Reply:
One or two of the tests have multiple inputs as I recall and this addresses that.

Reply:
Then, like for the other tests, you add a test-specific target.

Reply:
How about just test-specific dependencies and common target, like this?

Reply:
It looks like the other ilasm customizations are more custom. This is just additional source files and no other switches.

Reply:
Keep it consistent with the rest of the file

Reply:
I did, or there was no example -- if you only have ilasm and source files, this is the only case that looks like that. It only handled one source file, so extend it to multiple. The other cases that run ilasm all throw extra switches, so are not examples to closely follow. Downside might be GNU make specificity but we have that elsewhere.